src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,64 @@
+import * as Polymorphic from '@radix-ui/react-polymorphic'
+import React, { forwardRef } from 'react'
+import { CSS, StitchesVariants, styled } from 'stitches.config'
+import { Root, List, Tab as RadixTab, Panel } from '@radix-ui/react-tabs'
+import {
+  buttonBaseStyle,
+  mainVariants,
+  sizeVariants,
+  hover,
+  active,
+} from '../Button/Button'
+
+export const TabsRoot = styled(Root, {})
+
+export const TabsList = styled(List, {
+  display: 'flex',
+  borderBottom: '1px solid $grey4',
+  '& *:not(:first-child)': {
+    borderLeft: 'none!important',
+  },
+})
+
+const tabVariants = {
+  variant: mainVariants,
+  size: sizeVariants,
+  force: {
+    hover,
+    focus,
+    active,
+  },
+}
+
+const StyledTab = styled(RadixTab, {
+  ...buttonBaseStyle,
+  cursor: 'pointer',
+  minWidth: '160px',
+  borderRadius: '0',
+  border: 'none',
+
+  '&[data-state="active"]': {
+    boxShadow: 'inset 0 -1px 0 0 currentColor, 0 1px 0 0 currentColor',
+  },
+  variants: tabVariants,
+  defaultVariants: {
+    destructive: 'false',
+    size: 'default',
+  },
+} as Parameters<typeof styled>[1])
```

Comment:
Gosh, they are nasty types. 
To be fair they recommend not dismantling the styles in this way. 

They are working on the typescript support so maybe some nicer types will be published in the next release.